hip/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# VecMem project, part of the ACTS project (R&D line)
+#
+# (c) 2021 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# Enable HIP as a language.
+enable_language( HIP )
+
+# Project include(s).
+include(
+   "${CMAKE_CURRENT_SOURCE_DIR}/../cmake/vecmem-compiler-options-hip.cmake" )
```

Comment:
Is this preferable over modifying the cmake include search path?

Reply:
I was pondering about that myself as well... Maybe not...